code/modules/spells/roguetown/martialartist.dm

```diff
@@ -0,0 +1,212 @@
+// LORE : The Martial Artist has perfected a beautiful union of the arcyne with martial arts, and under Noc's gaze, he teaches the weak to defend themselves. 
+
+// some defines to make things work
+
+
+/mob/living/carbon/get_bodypart(zone)
+	if(!zone)
+		zone = BODY_ZONE_CHEST
+	for(var/obj/item/bodypart/bodypart as anything in bodyparts)
+		if(bodypart.body_zone == zone)
+			return bodypart
+		for(var/subzone in bodypart.subtargets)
+			if(subzone != zone)
+				continue
+			return bodypart
+
+// utility abilities
+
+/obj/effect/proc_holder/spell/self/ryusui
+	name = "Paws of Flowing Water"
+	desc = "Imbue your fists with the technique of the Flowing Water."
+	human_req = TRUE
+	miracle = FALSE
+	charge_max = 99999 MINUTES // one time self-buff.
+	invocation = "Paws of Flowing Water."
+	invocation_type = "none"
+	movement_interrupt = FALSE
+	sound = 'sound/magic/ryusuibuff.ogg'
+
+/obj/effect/proc_holder/spell/self/ryusui/cast(list/targets, mob/living/user)
+		user.change_stat("strength", 6)
+		user.change_stat("speed",3)
+		user.change_stat("perception",3)
+
+
+
+
+/obj/effect/proc_holder/spell/self/ryubreath
+	// pretty insane self heal, fickle to balance, i'm keeping it at 30 seconds for now.
+	name = "Steady Breath"
+	desc = "Steady your breathing, allowing the flowing waters to heal your wounds."
+	human_req = TRUE
+	miracle = FALSE
+	charge_max = 25 SECONDS
+	invocation = "Aaahh..."
+	invocation = "none"
+	movement_interrupt = FALSE
+
+
+/obj/effect/proc_holder/spell/self/ryubreath/cast(list/targets, mob/living/user)
+	user.visible_message(span_warning("[user] steadies their breath, the flowing water technique rejuvinating them slightly!"), span_notice("I steady my breath, rejuvinating myself..."))
+	user.adjustBruteLoss(-35)
+	user.adjustFireLoss(-25)
+	user.adjustToxLoss(-10)
+	user.blood_volume = min(user.blood_volume+90, BLOOD_VOLUME_NORMAL)
+	user.heal_wounds(35)
+	// heals most common damage types + gives some blood.
+
+// combat abilities
+
+
+/obj/effect/proc_holder/spell/invoked/ryunecksnap
+	name = "Neck Snap" // Instant kill ability. Don't abuse it, do some interesting RP first!!!
+	desc = "Snaps your opponent's neck. Very painful."
+	human_req = TRUE
+	range = 1
+	miracle = FALSE
+	charge_max = 40 SECONDS
+	invocation = "Heh."
+	invocation_type = "none"
+	movement_interrupt = FALSE
+
+/obj/effect/proc_holder/spell/invoked/ryunecksnap/cast(list/targets, mob/living/user)
+	if(isliving(targets[1]))
+		var/mob/living/carbon/target = targets[1]
+		var/obj/item/bodypart/BPN = target.get_bodypart(BODY_ZONE_PRECISE_NECK)
+		target.visible_message(span_warningbig("[target] has their neck suddenly snapped with a sickening sound!"), span_userdanger("Your vision suddenly darkens, you have trouble breathing..."))
```

Comment:
```suggestion
		target.visible_message(span_warningbig("[target] has [target.p_their()] neck suddenly snapped with a sickening sound!"), span_userdanger("Your vision suddenly darkens, you have trouble breathing..."))
```
I don't like the passive voice here ('target has their neck snapped'), I think it should really be something more like 'user swiftly snaps [target]'s neck with a sickening sound' or whatever.